components/list/list.css

```diff
@@ -0,0 +1,51 @@
+.list {
+  min-width: 10%;
+  list-style: none;
+  font-size: 1.2rem;
+  padding-inline-start: 0;
+}
+
+.list-title {
+  cursor: pointer;
+  padding: 0.5rem 0;
+  margin-bottom: 0.8rem;
+  border-bottom: 1px var(--black) dashed;
+  font-weight: 600;
+  font-size: 1.5rem;
+}
+
+.list-title:hover {
+  color: var(--secondary-color);
+}
+
+.list li {
+  cursor: pointer;
+  padding: 0.5rem;
+}
+
+.list li:hover {
+  color: var(--primary-color);
+}
+
+.list-sub ul {
+  list-style: none;
+  padding-inline-start: 0.5rem;
+}
+
+.list-sub ul li {
+  color: var(--black);
+}
+
+.check-list .check-title {
+  font-size: 1.4rem;
+  margin: 0.5rem 0;
```

Comment:
for sizes less than 1rem try to use px.

Reply:
0.5 rem is equal to 8 px which is not too much small to use pixels directly.